crates/services/txpool_v2/src/tests/tests_service.rs

```diff
@@ -236,6 +252,89 @@ async fn test_prune_transactions_the_oldest() {
     service.stop_and_await().await.unwrap();
 }
 
+#[tokio::test]
+async fn prune_expired_transactions() {
```

Comment:
can you add a test that checks if the `tx_status_sender` does not send duplicate tx statuses for transactions that may have been removed twice acc to my earlier comment?